lib/OpenQA/Schema/ResultSet/Assets.pm

```diff
@@ -73,44 +73,32 @@ sub scan_for_untracked_assets {
     my ($self) = @_;
 
     # search for new assets and register them
-    for my $type (qw(iso repo hdd)) {
-        my $dh;
-        next unless opendir($dh, $OpenQA::Utils::assetdir . "/$type");
-        my %assets;
+    for my $type (TYPES) {
         my @paths;
-        for my $file (readdir($dh)) {
-            unless ($file eq 'fixed' or $file eq '.' or $file eq '..') {
-                push(@paths, "$OpenQA::Utils::assetdir/$type/$file");
-            }
-        }
-        closedir($dh);
-        if (opendir($dh, $OpenQA::Utils::assetdir . "/$type" . "/fixed")) {
+
+        for my $subtype (qw(/ /fixed)) {
+            my $path = $OpenQA::Utils::assetdir . "/$type$subtype";
```

Comment:
You can also move `$OpenQA::Utils::assetdir` within the double quotes.

Reply:
Ah, good point!